go/enclave/container/test.toml

```diff
@@ -1,4 +1,4 @@
-hostID = "0x0000000000000000000000000000000000000000"
+hostID = "0x0000000000000000000000000000000000000001"
```

Comment:
I changed this so it doesn't clash with the sequencer ID, though it won't actually matter.